deps/build.jl

```diff
@@ -10,18 +10,20 @@ provides(AptGet, Dict("libgsl0ldbl"=>libgsl, "libgsl0-dev" =>libgsl, "gsl-bin"=>
 provides(Yum, "gsl-devel", libgsl)
 provides(Pacman, "gsl", libgsl)
 
-@osx_only begin
+@compat @static if is_apple() begin
```

Comment:
need to add Compat, with minimum version, to REQUIRE. don't need either compat or static macro here, but do need using Compat